driver/executor/executor.go

```diff
@@ -307,74 +307,6 @@ func scheduleNodeEvents(node *parser.Node, queue *eventQueue, net driver.Network
 
 		queue.add(nodeCreate)
 
-		// 2. Queue Timer SimEvents
```

Comment:
This removes the timer config, which is separate from validator config.

I understand the rationale for this change, but it would make sense to make a different PR and remove any usage of timer from any scenarios there.

Reply:
yes, I can put it in another PR

Reply:
I can handle the the timer config revert PR.

Reply:
ok, thanks